box_it_script/answer.js

```diff
@@ -0,0 +1,67 @@
+#!/home/nimbus-user/.nvm/versions/node/v17.6.0/bin/node
```

Comment:
did you branch out from `box_it_script`? this feature branch should not include any commits from homework 1.

Reply:
Yeah, forgot to checkout to main and then made a new branch, will do in next homework